druid/src/app.rs

```diff
@@ -91,6 +91,16 @@ impl<T: Data> AppLauncher<T> {
         self
     }
 
+    /// Paint colorful rectangles for layout debugging.
+    ///
+    /// The rectangles are the size of each widget's `PaintCtx`. The colors are
+    /// distinct per `WidgetPod`, so each non-`WidgetPod` child of a `WidgetPod`
+    /// will have the same color as its parent.
```

Comment:
I think this wording is also slightly confusing; and in addition I'm backing off the reuse of ids, so padding/align will be treated uniquely.

Given this, I think we can get away without mentioning `WidgetPod`.

Additionally, I'm pretty sure the colors aren't actually distinct per pod, but I don't think we need to mention that.